column/api/run.py

```diff
@@ -0,0 +1,28 @@
+import logging
+
+from flask import Flask
```

Comment:
Flask is compatible with WSGI, and can be deployed with apache2 and uwsgi.
http://flask.pocoo.org/docs/0.12/deploying/.
Built-in server is definitely for development and testing only, we need a dockerfile with apache or nginx instead.